.changeset/stupid-pears-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/view-components": patch
```

Comment:
this could be considered breaking, but I don't know that it actually needs to be, if we don't believe this package is used by consumers who don't have a transpile step

Reply:
We don't release major versions of PVC at the moment. I would bump the minor version for this change, since it's not necessarily backwards-compatible.

Reply:
Makes sense!  I went back and forth between them!  Updated